microsoft-edge/webview2/concepts/working-with-local-content.md

```diff
@@ -362,6 +362,16 @@ Local content that's loaded via virtual host name mapping has an HTTP or HTTPS U
 Virtual host name URLs are resolved in WebView2 processes. This is a faster option than `WebResourceRequested`, which resolves in the host app process UI thread.
 
 
+<!-- ---------- -->
+###### Source maps with virtual host name mapping
+
+Source maps are needed to debug the source code of compiled content like transpiled JavaScript (e.g. TypeScript, minified JavaScript) or CSS (e.g. SASS, SCSS). WebView2 does not load source maps referenced by content which was loaded using virtual host name mapping. Consider the following example. WebView2 loads JavaScript file `main.js` via virtual host name mapping. If `main.js` references `main.js.map` as its source map, then `main.js.map` will neither be loaded automatically nor any `WebResourceRequested` event handler will be called to load it.
+
+To use source maps along with virtual host name mapping, choose one of the following approaches:
+- Generate inline source maps during compilation of your content. Inline source maps are embedded to the corresponding compiled file.
+- Use `WebResourceRequested` event instead, and inline separate source maps to the content at runtime in your `WebResourceRequested` event handler. Use this approach only if your content build system does not support inline source maps.
```

Comment:
"inline separate source maps to the content at runtime in your WebResourceRequested event handler"
Can we explain with an example here?

Rendered:
* **Using local content in WebView2 apps** > **Source maps with virtual host name mapping**
   * `/webview2/concepts/working-with-local-content.md`
   * Internal preview: https://review.learn.microsoft.com/microsoft-edge/webview2/concepts/working-with-local-content?branch=pr-en-us-3384#source-maps-with-virtual-host-name-mapping
   * External preview: https://github.com/mikosg/edge-developer/blob/source_maps/microsoft-edge/webview2/concepts/working-with-local-content.md#source-maps-with-virtual-host-name-mapping

Reply:
Outdated b/c I reformatted section.
Fresh link pending